index.html

```diff
@@ -1,12 +1,256 @@
+      </div>
+      <nav class="toggle-nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="https://amrendrakind.github.io/conferencePage_capstoneProject/" class="nav-link active">Home</a>
+          </li>
+          <li class="nav-item">
+            <a href="about.html" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Join</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Sponsor</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">News</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link btn-link">CC Campaign</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+
+    <header class="wrap header-wrap">
+      <div class="content">
+        <a href="https://amrendrakind.github.io/conferencePage_capstoneProject/"
+          ><img
+            src="images/logo_CC_Global_Summit_2015.png"
+            alt="Global Summit "
+        /></a>
+        <nav>
+          <ul class="nav-list">
+            <li class="nav-item">
+              <a href="index.html" class="nav-link active">Home</a>
+            </li>
+            <li class="nav-item">
+              <a href="about.html" class="nav-link">About</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-link">Program</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-link">Join</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-link">Sponsor</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-link">News</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-link btn-link">CC Campaign</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <div class="main_invite_wrap wrap">
+      <div class="mobile_menu_button">
+          <img src="./images/mobile_menu_button.png" alt="mobile_menu"/>
+      </div>
+      <section class="content">
+        <div>
+          <img
+            src="./images/landingPageTitle.png"
+            alt="welcome sharing world!"
+          />
+        </div>
+        <div class="message">
+          <p>
+            A joyful celebration believing in the value of openness and sharing,
+            creating a positive change with people from over 80 countries is
+            taking place in October, in Korea.
+          </p>
+        </div>
+        <div class="location">
+          <div class="date-info">2015.10.15 (THU) ~ 16 (FRI)</div>
+
+          <div class="location-address">
+            @ National Museum of Korea, Art Center Nabi, and more
```

Comment:
- [ ] Please, consider using semantic tags on whole `index.html`, sections use `section` - you are using here only `div` tags for sections -  I know you can fix it easily 💯 👍